content/announcing-sigstore-go.md

```diff
@@ -0,0 +1,24 @@
++++
+title = "Announcing sigstore-go"
+author = "The Sigstore Technical Steering Committee"
+date = "2023-10-05"
+draft = false
+type = "post"
+tags = ["sigstore","go","clients"]
++++
+
+
+# Announcing sigstore-go
+
+Today we’re excited to announce a new open source library, [sigstore-go](https://github.com/sigstore/sigstore-go) that represents the future of Sigstore’s support for the Go programming language. Since the beginning, Sigstore has been primarily written in Go but there has been a gap over the past year or so since we established the [Protobufs-based bundle format](https://github.com/sigstore/protobuf-specs): the de facto standard client (Cosign) lacks support for it. sigstore-go represents the TSC’s vision for future integrations in Go and will be used to eventually make Cosign smaller and more performant in the future, as well as provide the basis for Sigstore bundle support in the [policy controller](https://github.com/sigstore/policy-controller).
```

Comment:
Is it accurate to say "more performant"? I wouldn't expect this refactor to significantly improve performance, unless you've done testing and see otherwise.